src/madetomeasure_api/db/core.clj

```diff
@@ -4,7 +4,7 @@
 
 (def db-spec
   {:subprotocol "postgresql"
-   :subname "//localhost/madetomeasure_api"
+   :subname "//localhost/madetomeasure_development?user=matthewkirk"
```

Comment:
Probably should stay away from yaml since SnakeYAML is annoying to use... I'll convert it to env vars